src/components/Reports/GoalCalculator/SharedComponents/GoalCalculatorGrid/GoalCalculatorGrid.tsx

```diff
@@ -232,36 +259,68 @@ const GoalCalculatorGridForm: React.FC<GoalCalculatorGridFormProps> = ({
   ];
 
   return (
-    <StyledCard>
-      <StyledControlsBox>
-        <StyledAddButton
-          variant="text"
-          onClick={addExpense}
-          size="small"
-          startIcon={<AddIcon />}
-        >
-          {t('Add {{category}}', { category: categoryName })}
-        </StyledAddButton>
-        <StyledToggleBox>
-          <Switch
-            checked={directInput}
-            onChange={handleDirectInputToggle}
+    <>
+      <StyledBox>
+        <Typography variant="h6" component="span" sx={{ mr: 2 }}>
+          {categoryName}
+        </Typography>
+        <StyledButtonGroup>
+          <StyledModeButton
```

Comment:
What do you think about dropping all the custom styles on the `Button`s and using `variant="contained"` for the active button? That makes it match our theme more closely. Most/all of the styles on the `ButtonGroup` don't seem to be needed either.